### PR TITLE
finish implementing pardon-ip

### PR DESCRIPTION
**Pull request by dschwartz783**
The last pardon-ip commit (which allows IP pardons without restarting https://github.com/iTXTech/Genisys/commit/f61db12657d85af7ddc8b21c98bbf272c44efe43) wasn't complete. I completed it. This has been tested to work as advertised.

### CODE REVIEW

File: src/raklib/server/ServerHandler.php [hunk 1]
```diff
@@ -56,6 +56,11 @@ public function blockAddress($address, $timeout){
         $this->server->pushMainToThreadPacket($buffer);
     }
 
+    public function unblockAddress($address){
+        $buffer = chr(RakLib::PACKET_UNBLOCK_ADDRESS) . chr(strlen($address)) . $address;
+	$this->server->pushMainToThreadPacket($buffer);
```
Comment (dktapps) [inline src/raklib/server/ServerHandler.php]: Please fix this formatting.

Comment (dschwartz783) [inline src/raklib/server/ServerHandler.php]: Whoops! Can't believe I missed that. Will do.

**Comment by dktapps** [comment]
The code looks okay, but please use tabs instead of spaces for indentation. Rejecting until this is fixed.

**Comment by dktapps** [comment]
You're going to have a lot of work; you've used spaces _everywhere_

**Comment by dschwartz783** [comment]
RakLib uses spaces... I'll fix it for the pocketmine parts.

**Comment by dktapps** [comment]
Leave RakLib. I'll reformat it at a later date. But yes, please keep the PM part as tabs.

**Comment by dschwartz783** [comment]
Should be good to go now. Let me know if there are any other issues.

**Comment by dktapps** [comment]
That looks better. Once you're done, please squash your commits by rebasing.

**Comment by dschwartz783** [comment]
not sure if I did that right... Srry, somewhat new to git

File: src/pocketmine/network/Network.php [hunk 2]
```diff
@@ -288,6 +288,35 @@ public function blockAddress($address, $timeout = 300) {
 		}
 	}
 
+	/**
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> f1d1bf79770b36307efb3e323c15a18a6f05ab8d
```
Comment (dktapps) [inline src/pocketmine/network/Network.php]: What's this?

**Comment by dktapps** [comment]
Here, this might help: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by dktapps** [comment]
And revert those last 2 commits before you do, I've no idea what you did there

**Comment by dschwartz783** [comment]
That should do it

**Comment by dktapps** [comment]
Much better. Code Review +1

[Approved with PullApprove](https://pullapprove.com/iTXTech/Genisys/pull-request/1294/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=iTXTech/Genisys)

**Comment by PeratX** [comment]
+1

**Pull request merged**